terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -742,13 +742,26 @@ def remove_inconsequential_statements(statements: List[JSON]) -> List[JSON]:
                 ]
             }
         },
+        "aws_s3_bucket": {
+            "gitlab": {
+                "bucket": f"edu-ucsc-gi-singlecell-hca-gitlab-{config.deployment_stage}",
```

Comment:
```suggestion
                "bucket": f"edu-ucsc-gi-singlecell-azul-gitlab-{config.deployment_stage}-{aws.account}",
```

Reply:
@hannes-ucsc : "Sorry, I meant `aws.region_name` instead of `aws.account`"